common/app/model/Asset.scala

```diff
@@ -107,8 +107,9 @@ case class VideoAsset(
   // The video duration in seconds
   val duration: Int = fields.get("durationSeconds").getOrElse("0").toInt +
                            (fields.get("durationMinutes").getOrElse("0").toInt * 60)
+  val durationSeconds: Int = fields.get("durationSeconds").getOrElse("0").toInt
```

Comment:
Sorry, I am being a fool, I didn't realise these were video specific.

You could make this slightly more readable by:
```
val durationSeconds: Int = fields.get("durationSeconds").map(_.toInt).getOrElse(0)
val durationMinutes: Int = fields.get("durationMinutes").map(_.toInt).getOrElse(0)
val duration: Int = durationSeconds + (durationMinutes * 60)
```

Saying that - why do we need `duration` in seconds?